Viewer.js

```diff
@@ -337,7 +337,8 @@ class MillicastWidget extends React.Component {
                   {this.state.playing ? 'Pause' : 'Play'}
                 </Text>
               </TouchableHighlight>
-              {this.state.playing ? (
+              {this.state.playing &&
+              !JSON.stringify(Platform).includes('AFTMM') ? (
```

Comment:
There are some other FireTV models that start with `AFT` but not `AFTMM`, it should include all possible values of FireTV.
Also, AndroidTV seems to be having the same issues, could it be disabled for that as well?

Reply:
Disabling this button for all AndroidTV-based devices (such as FireTV) would require a different approach.